setup.py

```diff
@@ -84,6 +84,7 @@
         extras_require={
             "test": [
                 "pytest>=3.6.0,!=3.9.0,!=3.9.1,!=3.9.2",
```

Comment:
Might as well bump pytest lower limit to at least 4.4 then since pytest-subtests doesn't run on any older version